gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/PresenterWidget.java

```diff
@@ -225,6 +226,10 @@ public void fireEvent(GwtEvent<?> event) {
         getEventBus().fireEventFromSource(event, this);
```

Comment:
OK I did the changes as requested (I did commit --amend to make sure that we don't pollute the commit history). Hope this is fine